website/docs/reference/manifest.md

```diff
@@ -70,6 +72,17 @@ edition = "{{ constants.edition }}"
 Most manifests have the edition field filled in automatically by `scarb new` with the latest available edition.
 If the edition field is not present in Scarb.toml, then the default edition is assumed.
 
+### `publish`
+
+The publish field is an optional key that determines whether the package can be packaged and published to a registry.
```

Comment:
```suggestion
The publish field is an optional key that determines whether the package can be published to a registry.
```

Reply:
`publish = false` disallows `scarb package` as well, as it was outlined in #636

Reply:
https://github.com/software-mansion/scarb/blob/295935d87b9731ea05839f7901f735bebf3d7060/scarb/tests/package.rs#L1331

We discussed it after merging initial impl and I changed it

Reply:
https://github.com/software-mansion/scarb/pull/1447

Reply:
Oh snap, totally forgot we changed this